src/rqt_console/console.py

```diff
@@ -100,7 +100,6 @@ def convert_rosgraph_log_message(log_msg):
         msg.message = log_msg.msg
         msg.severity = log_msg.level
         msg.node = log_msg.name
-        msg.stamp = (log_msg.header.stamp.sec, log_msg.header.stamp.nanosec)
```

Comment:
`log_msg` obtained from `/rosout` doesn't have `header` entity. So, I decided to remove this.